tests/ui/deprecated.stderr

```diff
@@ -78,11 +78,25 @@ error: lint `clippy::find_map` has been removed: this lint has been replaced by
 LL | #[warn(clippy::find_map)]
    |        ^^^^^^^^^^^^^^^^
 
+error: unknown lint: `clippy::filter_map`
```

Comment:
This should be a deprecation error. Not an unknown lint error.

Reply:
Thanks! I wasn't using `declare_deprecated_lint!`. Fixed. I also changed the message to be similar to `find_map`.